aicsimageio/readers/lif_reader.py

```diff
@@ -605,6 +605,22 @@ def _construct_mosaic_xarray(self, data: types.ArrayLike) -> xr.DataArray:
                 DimensionNames.SpatialX,
             ]
         }
+
+        # Add expanded Y and X coords
+        scale_x, scale_y, _, _ = selected_scene.info["scale"]
+        if scale_y is not None:
```

Comment:
if the scale values ARE none, does coords already contain the correct ranges?

Reply:
If the scale values are None, there are NO coord ranges for Y or X.

Reply:
Then I am missing some context here.  Intuitively, it would seem to me that missing scale is the same as scale=1

Reply:
This is one of the weird things about our Reader spec.

This is setting the physical pixel coords on the xarray obj. This is not setting the `physical_pixel_sizes` attr.

So the physical pixel sizes property will still return `<PhysicalPixelSizes Z=1, Y=1, X=1>` but the xarray object just wont have coords.

I originally made that decision because it was like "well we should have physical pixels sizes always available... but the coords are like... the real thing. we shouldn't do that?" idk.

Reply:
It's not a feature I would use anyway.  But it seems to me that I should be able to expect those coords to be there all the time.